tests/neo-vm.Tests/Tests/OpCodes/Push/PUSHA.json

```diff
@@ -0,0 +1,74 @@
+{
+  "category": "Push",
+  "name": "PUSHA",
+  "tests": [
+    {
+      "name": "Out of range [-1]",
+      "script": "0Affffffff",
```

Comment:
Same thing as before... looking too big this address, for general cases it will be certainly much smaller.